src/main/scala/com/codacy/client/bitbucket/v2/service/PullRequestServices.scala

```diff
@@ -17,8 +19,9 @@ class PullRequestServices(client: BitbucketClient) {
       repository: String,
       states: Seq[String] = Seq("OPEN")
   ): RequestResponse[Seq[PullRequest]] = {
-    val url =
-      s"https://bitbucket.org/api/2.0/repositories/$owner/$repository/pullrequests?pagelen=50&state=${states.mkString("&state=")}"
+    val pullRequestsUrl = generatePullRequestsUrl(owner, repository)
+    val encodedStates = states.map(state => URLEncoder.encode(state, "UTF-8"))
+    val url = s"$pullRequestsUrl?pagelen=50&state=${encodedStates.mkString("&state=")}"
```

Comment:
It's not really in scope, but maybe we can make the pagelen a parameter with the default 50?

Reply:
done